frontend/src/components/common/SimpleTable.tsx

```diff
@@ -72,37 +73,68 @@ export default function SimpleTable(props: SimpleTableProps) {
     emptyMessage = null,
     errorMessage = null,
   } = props;
+  const location = useLocation();
+  const history = useHistory();
   const [page, setPage] = React.useState(0);
   const [currentData, setCurrentData] = React.useState(data);
   const rowsPerPageOptions = props.rowsPerPage || [5, 10, 50];
   const [rowsPerPage, setRowsPerPage] = React.useState(rowsPerPageOptions[0]);
   const classes = useTableStyle();
 
+  function addQuery(queryObj:{[key:string]: string}) {
+    const pathname = location.pathname;
+    const searchParams = new URLSearchParams(location.search);
+    for (const key in queryObj) {
+      const value = queryObj[key];
+      if (value) {
+        searchParams.set(key, value);
+      } else {
+        searchParams.delete(key);
+      }
+    }
```

Comment:
I don't love noisy URLs. So I'd prefer if we only set these params if they differ from whatever we set as default.